packages/redis/src/mod.ts

```diff
@@ -35,9 +35,13 @@ export class RedisAdapter<T> implements StorageAdapter<T> {
   }
 
   async write(key: string, value: T) {
-    await this.redis.set(key, JSON.stringify(value));
-    if (this.ttl) {
-      this.redis.expire(key, this.ttl);
+    const val = JSON.stringify(value)
+    const ttl = this.ttl
+    
+    if (ttl) {
+      await this.redis.set(key, val, "EX", ttl);
+    } else {
+      await this.redis.set(key, val);
     }
```

Comment:
`ttl` would be `undefined` by default so we could just do this instead:


```suggestion
    await this.redis.set(key, val, "EX", ttl);
```

Reply:
Would you please add the same description you wrote as a comment above the line?
Just for clarity.